gnosis/safe-react
# Add EIP-3770 settings

## What it solves
Resolves #2843

## How this PR fixes it
- A new reducer has been added under the `appearance` key to store the new settings.
- A new settings subsection 'Appearance' has been added. It currently has EIP-3770 settings regarding showing/copying chain `shortName` prefixes.
- A checkbox has been added to the QR modals to allow for copying with/without `shortName`. It defaults to the value set in settings and does not update the store.

**Note:** the settings do not currently affect any of the UI except the QR modal.

## How to test it
Ensure that changes in the 'Appearance' settings save locally.
Open QR modal and check that the value reflects that saved.
Scan the QR code with/without the checkbox checked to see if `shortName` is prepended.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/138720139-7a9f7d20-fa85-4041-924a-0361b08ba086.png)
![image](https://user-images.githubusercontent.com/20442784/138702868-fb82cf55-f719-411b-b457-8a06b98e3d05.png)
